ProcessLib/ThermoMechanics/ThermoMechanicsProcess.cpp

```diff
@@ -54,6 +55,26 @@ ThermoMechanicsProcess<DisplacementDim>::ThermoMechanicsProcess(
                     result[i] = local_asm.getSigma();
                 }
 
+                return result;
+            }));
+
+    _integration_point_writer.emplace_back(
+        std::make_unique<SigmaIntegrationPointWriter>(
+            "epsilon_ip",
```

Comment:
I'd suggest to rename the SigmaIntegrationPointWriter in KelvinVectorIntegrationPointWriter; it looks strange to feed the sigma writer with epsilon_ip.

Reply:
done